src/components/Toast.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import elementStyles from '../styles/isomer-cms/Elements.module.scss';
+
+const Toast = ({ notificationType, text }) => {
+  const toastImg = () => {
+    switch(notificationType) {
+      case 'info':
+        return 'bxs-info-circle'
+      case 'success':
+        return 'bxs-check-circle'
+      case 'error':
+        return 'bxs-x-circle'
+      case 'warning':
+        return 'bxs-error'
+      default: 
+        return ''
+    }
+  }
+  return (
+    <div className={elementStyles.toastContent}>
+      <i className={`bx bx-sm ${toastImg()}`}/>
+      {text}
+    </div>
+  )
+}
+
+export default Toast
```

Comment:
Add proptypes for `Toast`

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/247/commits/8c58d2714674b84fb9f73524a16bf38166d198d6